pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -6,8 +6,10 @@ import (
 )
 
 var (
-	LabelBuild           = "build.build.dev/name"
-	LabelBuildGeneration = "build.build.dev/generation"
+	LabelBuild                 = "build.build.dev/name"
+	LabelBuildGeneration       = "build.build.dev/generation"
+	AnnotationBuildRunDeletion = "build.build.dev/buildRunDeletion"
```

Comment:
- About question 1:
I think the annotation or label should not just start as the api-group,
Like you see in Tekton:
```
- apiVersion: tekton.dev/v1beta1
  kind: TaskRun
  metadata:
    annotations:
      pipeline.tekton.dev/release: devel
```
Its annotations is `<function>.<api_group>/xxx`, we also follow this way, so that we can know this annotation is for build: `build.build.dev`, and another annotation is for buildrun: `buildrun.build.dev` easily

Build v1 in OpenShift is also different:
```
kind: Build
apiVersion: build.openshift.io/v1
metadata:
  annotations:
    openshift.io/build-config.name: cf-sample-app-go-git
    openshift.io/build.number: '2'
    openshift.io/build.pod-name: cf-sample-app-go-git-2-build
```

So if there is no special standard, let us follow the original way we use first.

- For question 2,
Yes, changed to `build.build.dev/build-run-deletion`

Reply:
for ur tekton example, then it should have been `taskrun.tekton.dev/release: devel` instead of `pipeline.tekton.dev/release: devel`. But this is not a blocker for this PR, so Im fine by just having the discussion, also thanks for removing the camelCase